core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/internal/ResourceDocument.java

```diff
@@ -60,6 +61,13 @@
 	}
 )
 @JsonDeserialize(builder = ResourceDocument.Builder.class)
+@Script(
+		name="typeSort", 
+		script=
+		 "if (params.orderByType.containsKey(doc.resourceType.value)) {"
+		+ "	return params.orderByType.get(doc.resourceType.value)"
+		+ "}"
+		+ "return '5'")
```

Comment:
The script should fall back to sorting by `doc.resourceType.value` if the map does not have an entry for it.

Reply:
Fixed.